fvm/src/syscalls/filecoin.rs

```diff
@@ -178,3 +178,15 @@ pub fn batch_verify_seals(
     }
     Ok(())
 }
+
+/// Returns the network circ supply split as two u64 ordered in little endian.
```

Comment:
You could drop the mention of "splitting into two u64"s here, that's obvious from the `sys::TokenAmount` type.